ingest/expression_files/dense_command.py

```diff
@@ -0,0 +1,30 @@
+import abc
+import sys
+
+from command import Command
+from expression_files.dense import Dense
+
+sys.path.append("../ingest")
```

Comment:
Typically this is put above certain imports, but given no imports are below it this line seems removable.

```suggestion
```

Reply:
This file was removed. Suggestion still applied in 
be99f49